docs/api/quality-attributes.md

```
@@ -1,130 +1,107 @@
-## Performance
+### Performance
 
-- **Target**: Список доступних паркомісць, історії оплат або парковки на карті мають відображатися ≤ 3 секунд після запиту
+- **Target**: The list of available parking spots, payment history, or parking map must be displayed within ≤ 3 seconds after a request.  
 - **Implementation**:
-    - Використовується ORM із `select_related()` у `SpotViewSet` та `BookingViewSet` для зменшення кількості SQL-запитів.
-    - Основні ендпоінти (`/lots/`, `/lots/{id}/spots/`, `/bookings/`) виконують прості фільтраційні операції без складних обчислень.
-    - Серіалізатори DRF формують оптимізовані відповіді без надлишкових вкладених структур.
-    - Вбудована пагінація скорочує обсяг даних у відповіді, що прискорює відображення списків.
-    - `select_related("lot")` використовується для попереднього завантаження пов’язаних даних про паркінг для кожного місця.
+    - ORM with `select_related()` is used in `SpotViewSet` and `BookingViewSet` to reduce the number of SQL queries.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)